### ISSUE TITLE
[BUG] In "speak" ENTER shortcut does not work and resets recordings

### ISSUE THREAD

**Issue by HarikalarKutusu** [common-voice/common-voice#4555]
This came up in matrix (starts [here](https://matrix.to/#/!WpvTgzNfcLIgzJZdSE:mozilla.org/$4iwSNJ29zC7e6nW7mclPlxW6thrnVWaXiWvLsIhjxRM?via=mozilla.org&via=matrix.org&via=t2bot.io) and continues).

The keyboard shortcuts in `speak` indicate that you can use `ENTER` to post recorded 5 batch, and it was working before.
A while ago a new popup has been added to accept the MCV stuff:
![image](https://github.com/user-attachments/assets/092b2c29-dee0-4f7d-9052-895c1cf02d17)

And now you cannot use the `ENTER` key here, worse, it resets the 5 recordings you already made.

**Expected behavior**
ENTER can be used in the "pop-up" like before

Alternative (quick): Remove `ENTER` from shortcuts.

**Comment by moz-kathyreid** [comment]
I confirmed this is still occurring as of release 137.7

**Comment by HarikalarKutusu** [comment]
I have a hard time debugging this. The whole contribution are, wrapped with `contribution.txt` makes it hard to follow.
There are also too many branches/states to test, such as being logged in/out or accepted some item or not.

I need some team support in this (pinging @moz-rotimib).

During debugging (logged-in) it errors out in this line, because there is no event passed to the handler.
https://github.com/common-voice/common-voice/blob/f2eac38c70c4eb491d6d2e186fbfb365e8e0e20c/web/src/components/pages/contribution/speak/speak.tsx#L531

I debug it like this getting NO EVENT and it works when logged-in and with the addition below:
```
    if (evt) {
      console.log("EVENT:", evt)
      evt.preventDefault()
    } else {
      console.log("NO EVENT")
    }
```

I had to add ``autoFocus to the Submit button for Enter to work correctly, although not advised by a11y:
```
                <Localized id="submit-form-action">
                  <PrimaryButton
                    className={[
                      'submit',
                      getTrackClass('fs', `submit-${type}`),
                    ].join(' ')}
                    disabled={!this.isDone}
                    type="submit"
                    data-testid="submit-button"
                    // eslint-disable-next-line jsx-a11y/no-autofocus
                    autoFocus={true}                                                                            // <== ADDED
                  />
                </Localized>
```
That button is here:
https://github.com/common-voice/common-voice/blob/f2eac38c70c4eb491d6d2e186fbfb365e8e0e20c/web/src/components/pages/contribution/contribution.tsx#L599

**Comment by moz-kathyreid** [comment]
I am working on another Issue and am using keyboard shortcuts and came up against this Issue at the same time. Keep seeing this error in the Console if I use the Enter key to Submit, logging so we have more complete information. 

---

```
Uncaught TypeError: can't access property "preventDefault", evt is undefined
    handleSubmit speak.tsx:407
    handleKeyDown contribution.tsx:79
    componentDidMount contribution.tsx:96
    React 6
    unstable_runWithPriority scheduler.development.js:468
    React 4
    unstable_runWithPriority scheduler.development.js:468
    React 2
    workLoop scheduler.development.js:417
    flushWork scheduler.development.js:390
    performWorkUntilDeadline scheduler.development.js:157
    EventHandlerNonNull* scheduler.development.js:180
    <anonymous> scheduler.development.js:645
    57409 vendors-node_modules_r.c85f851ac1c202e9.js:50
    __webpack_require__ runtime.eb8a46b73d9c251d.js:31
    <anonymous> index.js:6
    91434 vendors-node_modules_r.c85f851ac1c202e9.js:61
    __webpack_require__ runtime.eb8a46b73d9c251d.js:31
    React 2
    96737 vendors-node_modules_react-dom_cjs_react-dom_development_js-f8c1fc6b.9430d10507c44878.js:18
    __webpack_require__ runtime.eb8a46b73d9c251d.js:31
    <anonymous> React
    78325 vendors-node_modules_react-b.9ea4de28e2717001.js:50
    __webpack_require__ runtime.eb8a46b73d9c251d.js:31
    <anonymous> 1863:5
    1863 main-src_c.ab140cadaf59bed5.js:84
    __webpack_require__ runtime.eb8a46b73d9c251d.js:31
    __webpack_exec__ main-src_s.96a4e08ff7327b3b.js:194
    <anonymous> main-src_s.96a4e08ff7327b3b.js:195
    O runtime.eb8a46b73d9c251d.js:68
    <anonymous> main-src_s.96a4e08ff7327b3b.js:196
    webpackJsonpCallback runtime.eb8a46b73d9c251d.js:306
    <anonymous> main-src_s.96a4e08ff7327b3b.js:10
speak.tsx:407

```

**Comment by moz-bozden** [comment]
This has been fixed in https://github.com/common-voice/common-voice/pull/5236
